<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APKFilesViewer</class>
 <widget class="QDialog" name="APKFilesViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>628</width>
    <height>616</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>616</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APK Files Viewer</string>
  </property>
  <widget class="QToolButton" name="tbOpen">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open (Ctrl+A)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>71</y>
     <width>608</width>
     <height>511</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::NoDragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::IgnoreAction</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <property name="movement">
    <enum>QListView::Static</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::Batched</enum>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="viewMode">
    <enum>QListView::IconMode</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tbReload">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Reload (F5)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/reload.png</normaloff>:/img/reload.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ePath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>45</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lDir">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>45</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dir:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tbMiPC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>PC Drives</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/pc.png</normaloff>:/img/pc.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Backspace, Return</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/process.png</normaloff>:/img/process.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lCount">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 150);
border: 5px; border-radius:20px;
color:white;
font: bold 10pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tbView">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Chage View</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/eye.png</normaloff>:/img/eye.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Alt+V</string>
   </property>
  </widget>
  <widget class="QLabel" name="ltext">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>78</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(235,235,235);
background-color: rgb(16, 96, 194);
border-color: rgb(255,255, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="tbOrderAZ">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="toolTip">
    <string>Order A-Z</string>
   </property>
   <property name="text">
    <string>A-Z</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbUp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>40</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Up (BackSpace)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/up.png</normaloff>:/img/up.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbSelectAll">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>45</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select All (F4)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/check</normaloff>:/img/check</iconset>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbSelectAll_2">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>45</y>
     <width>25</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show in Explorer (Ctrl + Alt + E)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/explorer.png</normaloff>:/img/explorer.png</iconset>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </widget>
  <action name="actionRename">
   <property name="text">
    <string>Rename to Recomended</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDelete_All_Selected">
   <property name="text">
    <string>Delete All Selected</string>
   </property>
  </action>
  <zorder>tbOpen</zorder>
  <zorder>label</zorder>
  <zorder>tbReload</zorder>
  <zorder>ePath</zorder>
  <zorder>lDir</zorder>
  <zorder>tbMiPC</zorder>
  <zorder>tbRename</zorder>
  <zorder>listWidget</zorder>
  <zorder>lCount</zorder>
  <zorder>tbView</zorder>
  <zorder>ltext</zorder>
  <zorder>tbOrderAZ</zorder>
  <zorder>tbUp</zorder>
  <zorder>tbSelectAll</zorder>
  <zorder>tbSelectAll_2</zorder>
 </widget>
 <resources>
  <include location="check.qrc"/>
 </resources>
 <connections/>
</ui>
